Contextual Help ownCloud Web (#88)
owncloud/docs-webui: modules/owncloud_web/pages/web_for_users.adoc

// FIXME: Elaborate more when it's working. Is there a diff to come? I hope so.

== Contextual Help

> ownCloud Web offers contextual help. In those cases where possible, you will see a questionmark as shown in the image below:

Reviewer: ```suggestion
ownCloud Web offers contextual help. If available, you will see a question mark as shown in the image below:
```

image::web-for-users/share-via-link.png[Sharing,width=400]

> When clicking on the questionmark, a context specific frame with additional help text shows up which can be closed by clicking outside, see the image below for an example.

Reviewer: ```suggestion
When clicking on the question mark, a context specific frame providing a help text will show up and can be closed by clicking outside outside the frame. See the image below for an example.
```
